### ISSUE TITLE
Error in 'Generate feed' not informative enough

### ISSUE THREAD

**Issue by VirginiaBalseiro** [dokieli/dokieli#512]
When filling in the ‘Generate feed’ form, an error is displayed, but it does not clearly indicate the location where the mistake was made without using text.

In the example below, the URL field should be highlighted, and more info should be given about how to fix the error (such as offering an example).

<img width="826" height="653" alt="Image" src="https://github.com/user-attachments/assets/f06706c6-6223-4af9-b50f-66ded6996991" />

See: https://www.w3.org/WAI/WCAG22/Understanding/error-identification
